docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -68,99 +73,111 @@ To develop and test the app, you'll install it on your [sandbox store](#configur
    
-   c. Under **Get started with Stripe**, copy the **Secret key**. In a later step, you will use the secret key to update the `STRIPE_SECRET_KEY` environment variable in the .env file.
+   c. Under **Get started with Stripe**, copy the **Secret key** and make sure to store it in a safe place. In a later step, you will use the secret key to update the `STRIPE_SECRET_KEY` environment variable in the .env file.
 
 ### Enable Stripe Connect for Platforms
 
-**During this instruction sequence, make sure that the left side of your Stripe Dashboard's top menu bar indicates that you're in the app-specific account.**
+**During this following steps, make sure that you are logged into your the app-specific account by ehcking the top left corner of your Stripe Dashboard.**
 
 <!-- theme: info -->
 > #### Test mode
-> You can simulate transactions in test mode to confirm your integration works correctly; the following steps are configured for test mode.
+> You can simulate transactions in test mode to confirm your integration works correctly. 
 
-1. Click the **Connect** button to enable [Stripe Connect for Platforms](https://dashboard.stripe.com/test/connect/accounts/overview). 
+**To configure test mode**
+
+1. On your Stripe Dashboard click **Connect** to enable [Stripe Connect for Platforms](https://dashboard.stripe.com/test/connect/accounts/overview). 
 
 2. Click **Get started**.
 
-3. Select **Platform or Marketplace**, then click the **Continue** button at the lower right.
+3. Select **Platform or Marketplace**, then click **Continue**.
+
+4. To configure the app-specific account's Stripe Connect settings, on your Stripe Dashboard go to [**Settings > Connect settings**](https://dashboard.stripe.com/test/settings/connect). 
 
-4. To configure the app-specific account's Stripe Connect settings, go to [**Settings > Connect settings**](https://dashboard.stripe.com/test/settings/connect). Under **Integration** perform the following steps:
+5. Under **Integration** do the following:
    
-   a. Copy the **Test mode client ID**. In a later step, you will use the client ID to update `NEXT_PUBLIC_STRIPE_CLIENT_ID` in the .env file.
+   a. Copy the **Test mode client ID** and make sure to store it at a safe place. You will use the client ID in a later step to update `NEXT_PUBLIC_STRIPE_CLIENT_ID` in the .env file.
 
    b. Set **OAuth settings** to **OAuth for Standard accounts**. 
 
-   c. Add **Redirects**. Click the **+ Add URI** button and add your app's callback URI.
+   c. Click **Redirects**, and then click **+ Add URI**. Add the callback URI for your app:
    
    ```http title="Your app's Stripe callback URI"
    https://{ngrok_id}.ngrok.io/stripe/callback
    ```
 
 ## Configure the BigCommerce store
 
-To configure the store to make subscription charges, take the following steps:
+After you have successfully created a test configuration, you need to configure your BigCommerce store in the BigCommerce store control panel.
```

Comment:
```suggestion
After you successfully create a test configuration, configure your BigCommerce sandbox store in the store control panel.
```

Reply:
what does `test configuration` mean here? It's vague. Be specific re what we accomplished in the preceding section.